src/tensorlake/documentai/_datasets.py

```diff
@@ -416,6 +417,112 @@ async def update_dataset_async(
         resp = await self._arequest("PUT", f"/datasets/{dataset.dataset_id}", json=body)
         return Dataset.model_validate(resp.json())
 
+    def get_dataset_data(
+        self,
+        dataset: Dataset,
+        cursor: Optional[str] = None,
+        direction: Optional[PaginationDirection] = None,
+        dataset_name: Optional[str] = None,
+        limit: Optional[int] = None,
+        file_name: Optional[str] = None,
+        status: Optional[ParseStatus] = None,
```

Comment:
Same here, looks like these are all filters. May be create a filter object to make this clear?